KlaviyoSwift.podspec

```diff
@@ -13,7 +13,7 @@ Pod::Spec.new do |s|
   s.source           = { :git => "https://github.com/klaviyo/klaviyo-swift-sdk.git", :tag => s.version.to_s }
   s.swift_version = '5.7'
   s.platform = :ios
-  s.ios.deployment_target = '13.0'
+  s.ios.deployment_target = '14.0'
```

Comment:
> If this is standard practice then we may as well do it in a major release.
> I do have slight reservation that we aren't really doing it for a positive reason like to attain new features, which makes it hard to justify to a developer who maybe has an app that still supports older OSes. But perhaps that is a very small/non-existent segment.

If this is true, we'd lose 0.7% of devices
https://iosref.com/ios-usage

Reply:
@evan-masseau

Reply:
Had a discussion internally and decided to leave this at 13 for now.